backend/functions/lib/adapters/gateways/dxtr/mock.offices.gateway.ts

```diff
@@ -19,4 +19,12 @@ export class MockOfficesGateway implements OfficesGatewayInterface {
   getOffices(_applicationContext: ApplicationContext): Promise<OfficeDetails[]> {
     return Promise.resolve(MockData.getOffices());
   }
+
+  async getOfficeByCourtIdAndOfficeCode(
+    _applicationContext: ApplicationContext,
+    courtId: string,
+    officeCode: string,
+  ): Promise<OfficeDetails> {
+    return MockData.getOfficeByCourtIdAndOfficeCode(courtId, officeCode);
+  }
```

Comment:
Interesting.  This is not exported and I don't see it used here in this module.  Why did we add this?